### PR TITLE
Website footer implementation

### PR DESCRIPTION
**Pull request by Tobipeters**

### CODE REVIEW

File: src/images/svg/index.tsx
Comment (Oluwa-nifemi) [inline src/images/svg/index.tsx:1]: Hey
Is there a reason we're using a specific file with the svgs?
You can just import the svgs as react components I've setup the nuxt.config to work fine for that

File: yarn.lock
Comment (Oluwa-nifemi) [inline yarn.lock:1]: Can you delete this please so that we can stick with npm

**Review by Oluwa-nifemi — COMMENTED** [review]
Can I get some details on the index.tsx file?

File: src/images/svg/index.tsx
Comment (Tobipeters) [reply src/images/svg/index.tsx:1]: No big reason, I can resolve to your convention for uniformity.

Comment (Oluwa-nifemi) [reply src/images/svg/index.tsx:1]: Yes please thanks for understanding

**Review by Oluwa-nifemi — APPROVED** [review]

**Pull request merged**